src/org/opendatakit/briefcase/ui/export/components/ConfigurationPanelForm.java

```diff
@@ -89,26 +90,21 @@
         buildPemFileDialog().choose().ifPresent(file -> setPemFile(Paths.get(file.toURI())))
     );
     pemFileClearButton.addActionListener(__ -> clearPemFile());
+
     startDatePicker.addDateChangeListener(event -> {
-      if (!isDateRangeValid()) {
-        showError(MessageStrings.INVALID_DATE_RANGE_MESSAGE, "Export configuration error");
-        startDatePicker.clear();
-      } else
-        onSelectStartDateCallbacks.forEach(consumer -> consumer.accept(extractDate(event)));
-    });
-    endDatePicker.addDateChangeListener(event -> {
-      if (!isDateRangeValid()) {
-        showError(MessageStrings.INVALID_DATE_RANGE_MESSAGE, "Export configuration error");
+
+      if (event.getNewDate() == null || isInvalidStartDate(extractDate(event)))
         endDatePicker.clear();
-      } else
-        onSelectEndDateCallbacks.forEach(consumer -> consumer.accept(extractDate(event)));
+
+      endDatePicker.setEnabled(event.getNewDate() != null);
```

Comment:
If you're up for that, all these `null` checks could be replaced by using a little bit of `Optional<LocalDate> startDate = Optional.ofNullable(event.getNewDate())`, but we can do that on a future PR as well.

Reply:
I guess a new PR for that would be apt since it is a different issue :)